app/presenters/scanned_book_show_presenter.rb

```diff
@@ -3,8 +3,11 @@ class ScannedBookShowPresenter < CurationConcerns::GenericWorkShowPresenter
 
   def file_presenters
     @generic_files ||= begin
```

Comment:
Oh, I usually do it in this case so that the @generic_files ||= is scoped to multiple lines, so none of it will run if the instance variable is cached. Equivalent would be...

```ruby
def file_presenters
  return @generic_files if @generic_files
  ids = solr_document.fetch('generic_file_ids_ssim', [])
  @generic_files = CurationConcerns::PresenterFactory.build_presenters(generic_file_ids, GenericFilePresenter, current_ability)
end
```

In this refactor I could have removed it entirely - missed that fact.